### ISSUE TITLE
Add countdown placeholder

### ISSUE THREAD

**Issue by Minecrell** [Minecrell/ServerListPlus#103]
This was requested several times and would basically display the remaining time until a fixed date

**Comment by checkplys** [comment]
+1

**Comment by nick2017-Spigot** [comment]
+1

**Comment by nick-2017** [comment]
Please 4.0

**Comment by Minecrell** [comment]
It will be in 4.0.

**Comment by BrettplayMC** [comment]
+1
ETA of 4.0?

**Comment by Flavourized** [comment]
+1

@Minecrell 4.0 release coming in the next month?

**Comment by Minecrell** [comment]
@Flavourized No, 4.0 will not be coming in next month. See https://www.spigotmc.org/threads/serverlistplus.7371/page-75#post-2942270

**Comment by Flavourized** [comment]
No worries, was just curious since version is at 3.9.6 :p that is definitely understandable

**Comment by Tanimsiz** [comment]
is it added ? how can i use this placeholder ?

**Comment by Minecrell** [comment]
I'm looking for additional input for the "countdown" placeholder. What should they do precisely and how would the placeholders be used? What would it display?

For example, there are many different ways to display countdowns:  "5 days, 10 hours, ...", or "10:20:30" for hours:minutes:seconds etc

**Comment by kennyrkun** [comment]
let us pick in the configuration?

**Comment by Flavourized** [comment]
In the config you can add something like "countdown-format" and use placeholders like %y for years, %m for months, %d for days, %h, %m, %s. I'm sure you understand what I'm getting at. 

You can also include a whole countdown variable section where players can set their own custom (multiple) variables and set their own format for each
